README.md

```diff
@@ -72,13 +71,14 @@ from iopipe.iopipe import IOpipe
 def handler(event, context):
   iopipe = IOpipe(CLIENT_ID)
   timestamp = time.time()
+  iopipe.start_report(timestamp, context)
```

Comment:
Thinking more, it wouldn't be able to be `iopipe.Report()` but something more like:

```
report = iopipe.create_report(timestamp, context)
report.send()
```

Even better if it supports 'with'...

```
with iopipe.create_report(timestamp, context):
  # do stuff here
  pass
```